### PR TITLE
Perform renames towards "kategory" as the lib name

### PR DESCRIPTION
**Pull request by wiyarmir**
s/katz/kategory/

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ buildscript {
         gradleVersion = '3.5'
         gradleVersionsPluginVersion = '0.15.0'
         javaVersion = '1.7'
-        katzVersion = '0.1-SNAPSHOT'
+        kategoryVersion = '0.1-SNAPSHOT'
```
Comment (ffgiraldez) [inline build.gradle]: we probably need to update here to 0.3.3-SNAPSHOT

Comment (wiyarmir) [inline build.gradle]: how have been releases numbered so far? just by manually tagging?

Comment (wiyarmir) [inline build.gradle]: sorry, I was thinking to extract version to gradle.properties as many maven exported projects do, but that's probably subject for another PR ;)

**Comment by raulraja** [comment]
kategory.io has been registered and we can map it to the doc site as soon as there is one.

**Comment by wiyarmir** [comment]
(closing and reopening to trigger travis build)

**Pull request merged**